influxdb/src/main/scala/akka/stream/alpakka/influxdb/javadsl/InfluxDBFlow.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.influxdb.javadsl
+
+import akka.NotUsed
+import akka.stream.alpakka.influxdb.{InfluxDBSettings, InfluxDBWriteMessage, InfluxDBWriteResult}
+import org.influxdb.InfluxDB
+
+object InfluxDBFlow {
+
+  def create[T](
+      settings: InfluxDBSettings,
+      influxDB: InfluxDB
+  ): akka.stream.javadsl.Flow[InfluxDBWriteMessage[T, NotUsed], InfluxDBWriteResult[T, NotUsed], NotUsed] =
+    akka.stream.alpakka.influxdb.scaladsl.InfluxDBFlow.create(settings)(influxDB).asJava
```

Comment:
Could clean a lot of these types up by importing them. Would improve readability.